docassemble/ALDashboard/aldashboard.py

```diff
@@ -241,6 +246,35 @@ def init(self, *pargs, **kwargs):
         self.template_name = kwargs.get("template_name", "unknown_error")
 
 
+def install_fonts(the_font_files: DAFileList):
+    """
+    Install fonts to the server and restart both supervisor and unoconv.
+    """
+    # create the /var/www/.fonts directory if it doesn't exist
+    if not os.path.exists("/var/www/.fonts"):
+        os.makedirs("/var/www/.fonts")
+    
+    # save the DAFile to /var/www/.fonts
+    for f in the_font_files:
+        shutil.copyfile(f.path(), "/var/www/.fonts/" + f.filename)
```

Comment:
I'd be fairly worried about some sort of injection attack here (even though it's an admin only interview, idk). DA already uses `werkzeug.secure_filename()`, that should be a drop-in fix. (Might also have to import it in addition to this edit). 

```suggestion
        shutil.copyfile(f.path(), "/var/www/.fonts/" + werkzeug.secure_filename(f.filename))
```